packages/gatsby-theme-docs/src/overrides/page-header-banner-area.js

```diff
@@ -1,8 +1,5 @@
 import React from 'react';
-import UserResearchBanner from '../components/user-research-banner';
 
 // A banner to be rendered in the top-right corner of a content page
-const PageHeaderBannerArea = () => {
-  return <UserResearchBanner />;
```

Comment:
Removing the user research banner since we don't need it anymore. We removed it in the docs repo a while ago already